course_content/notebooks/dask_intro.ipynb

```diff
@@ -4,9 +4,45 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# An Introduction to Dask\n",
+    "# An Introduction to Dask"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Course overview\n",
+    "\n",
+    "Dask is a flexible parallel computing library for Python. It allows dynamic scheduling of tasks for optimising computations and includes collections (extensions of arrays, lists and dataframes) for use with parallel data processing.\n",
+    "\n",
+    "As of Iris v2.0, dask has been integrated with Iris. This means you can utilise all of the benefits of dask while keeping the fileformat interoperability and other functionality that Iris provides."
```

Comment:
@dkillick Minor ... but are we using `dask` or `Dask` ... there is a mixture of capitalization used throughout this document.

Reply:
We should be using however dask references itself. However, I've never managed to figure out what that is, so I've never been able to choose one to enforce!

Reply:
Right, I decided on "dask". I've implemented that decision throughout the notebook.